lib/internal/process/next_tick.js

```diff
@@ -5,6 +5,7 @@ exports.setup = setupNextTick;
 function setupNextTick() {
   const promises = require('internal/process/promises');
   const emitPendingUnhandledRejections = promises.setup(scheduleMicrotasks);
+  const MAX_TICKS = 1e4;
```

Comment:
If I’m reading the code correctly this should be the maximum number of callbacks that are run in a single event loop iteration – is that correct? Maybe there is a better name than `MAX_TICKS`, or maybe there should be a comment explaining what this does…